controller/array_action/array_mediator_ds8k.py

```diff
@@ -444,9 +444,8 @@ def unmap_volume(self, volume_id, host_name):
 
     def _get_api_volume_from_volumes(self, volume_candidates, volume_name):
         for volume in volume_candidates:
-            logger.info("Checking volume: {}".format(volume.name))
```

Comment:
should we log the length of `volume_candidates` somewhere?
it might be useful for performance analysis (e.g. relation between `get_volumes_by_pool` execution time and number of volumes in pool)